tests/manage/z_cluster/nodes/test_nodes_maintenance.py

```diff
@@ -439,6 +440,8 @@ def test_simultaneous_drain_of_two_ocs_nodes(
         self.sanity_helpers.health_check()
 
     @bugzilla("1861104")
+    @bugzilla("1946573")
+    @pytest.mark.polarion_id("OCS-2634")
```

Comment:
this will probably cause an override of one test case over the other in reporting to Polarion.
IMO, we don't need a new test case, but rather enhancing the existing one